app/helpers/application_helper.rb

```diff
@@ -166,7 +166,8 @@ def responsibles_selector_input(form)
     users = User.active.pluck(:id, :name, :username, :email).map { |user| [user[0], user[1].blank? ? user[2] : user[1], user[2], user[3]] }.sort_by { |user| user[1].downcase }
     form.input :responsibles, as: :select,
                               collection: options_for_select(users.map {|user| ["#{user[1]} (#{user[2]}) #{user[3]}", user[0]]}, @booth.responsibles.map(&:id)),
-                              include_blank: false, label: 'Responsibles', input_html: { class: 'select-help-toggle', multiple: 'true' }
+                              include_blank: false, label: 'Responsibles', input_html: { class: 'select-help-toggle', multiple: 'true' },
+                              hint: 'Only existing users can be assigned responsible for the booth'
```

Comment:
'can be assigned responsible' does not sound that correct.

What about this?
`The people responsible for the booth. You can only select existing users.`

Reply:
ok i will add this

Reply:
Please do :)

Reply:
Sorry for the delay.